components/UI/Map/VectorMap/DtpSource.tsx

```diff
@@ -62,7 +66,7 @@ export function DtpSource() {
         type: 'circle',
         source: 'ekb-dtp-source',
         paint: {
-            'circle-radius': 8,
+            'circle-radius': getLayerActiveStyle<number>(8, 10),
```

Comment:
probably better to use objects in the args of getLayerActiveStyle<number>({initial: 8, active: 10}). Just for clarity

Reply:
oh. and if you set default initial value, we can rename `getLayerActiveStyle` to `getLayerStyle`

Reply:
Good. I renamed.

In the future we will make `{initial, hover, active}`